### DISCUSSION TITLE
Why Spotube always use my location ? ( Windows )

### DISCUSSION THREAD

**Discussion by Thomas16093** [KRTirtho/spotube#899] [Q&A]
Hi, so first, thank you for this app, it's really great for listening albums on PC.
But, I don't understand why Spotube need my location.
If my location is allowed to the desktop applications, Spotube always use it ( via the windows indicator and in the settings ).

As stated In the presentation of this app, It shouldn't collect any data.
And, If I block all application for using my location, Spotube isn't bothered at first glance.

Is this utilisation normal ?

**Comment by KRTirtho** [comment]
What?

**Reply by KRTirtho** [reply]
There's literally not a single line of code on Spotube that uses Windows Location API 😅

**Reply by KRTirtho** [reply]
Are you sure it's Spotube? Maybe something else is using Location. Also, the indicator can indicate any apps using it.
If spotube uses Windows Location, it'll show up in the Location settings. Can you show that?

**Reply by KRTirtho** [reply]
Usually, browsers and sites like Google uses it. Close all your apps (even the background apps using task manager) and then run only Spotube and see if it shows up again

**Reply by Thomas16093** [reply]
Hi, thank you for the quick answer.
I really don't know why it use the location.
Here is what shown in my location settings ( sorry for the french language ) : 
![image](https://github.com/KRTirtho/spotube/assets/34454686/e22630d8-eb9f-4077-b553-354a6b59087e)
For some precision : 
I'm in the section that allow the desktop applications to use my location.
The text in red is : in use 

That happens on this PC with Windows 10 but also on my second PC with Windows 11, so I don't think it's just a bug on my PC.
Maybe you do need to explicitly refuse the location in your code...

**Comment by Augus1999** [comment]
I have the same question. The application keeps using location information on my Windows 11 laptop, even when it is running in the background.
